databricks/koalas/frame.py

```diff
@@ -655,6 +663,7 @@ def __setitem__(self, key, value):
         else:
             kdf = self.assign(**{key: value})
 
+        # Spark DataFrame
         self._sdf = kdf._sdf
```

Comment:
Can you do a type annotation instead? Either as a comment:
```
self._sdf = kdf._sdf  # type: pyspark.sql.DataFrame
```
or as an annotation:
```
self._sdf: pyspark.sql.DataFrame = kdf._sdf
```
Both should be picked up by mypy